packages/react-components/react-search/stories/SearchBox/SearchBoxUncontrolled.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { SearchBox } from '@fluentui/react-search';
+import { Field } from '@fluentui/react-components';
+import type { SearchBoxProps } from '@fluentui/react-search';
```

Comment:
It's up to you, but I don't think you really need this as a separate example, since the default example is already uncontrolled.